pep-0703.rst

```diff
@@ -1349,22 +1474,41 @@ be able to.
 Performance
 ===========
 
-The changes to make CPython thread-safe without the GIL have a
-negative performance impact on single-threaded performance.  The
-largest impact is due to the reference counting changes, particularly
-biased reference counting and immortalization. On Python 3.11,
-implementing biased reference counting and immortalization results
-in about a 10% geomean regression on the pyperformance suite.  This
-performance impact can be partly mitigated through further interpreter
-changes.  For example, with the "nogil" proof-of-concept [#nogil]_,
-biased reference counting and immortalization together have a smaller
-5% regression on the pyperformance suite.  However, those changes
-are not part of this PEP.
+The changes to make CPython thread-safe without the GIL have a small
+negative impact on single-threaded performance.  The largest impact is
+due to the reference counting changes, particularly biased reference
+counting.  The reference implementation contains some optimizations for
+programs that only use one thread.  The table below therefore breaks
+down execution overhead by single and multi-threaded programs.
+
+
+.. list-table:: Execution Overhead on pyperformance 1.0.6
+   :header-rows: 1
+   :widths: auto
+
+   * -
+     - Intel Skylake
+     - AMD Zen 3
+   * - One thread
+     - 6%
+     - 5%
+   * - Multiple threads
+     - 8%
+     - 7%
```

Comment:
Perhaps I'm not understanding, but you outline that the single-thread case has worse performance, and then show a lower single-threaded overhead in the table, indicating that multi-threaded performance is worse?

Reply:
That's clearer -- thank you!